src/adv.py

```diff
@@ -36,11 +47,55 @@
 #
 # Main
 #
-
 # Make a new player object that is currently in the 'outside' room.
 
+# player = Player('Peter Pan', room['outside'])
+clearScr()
+player_name = input("Please enter your name: ")
+
+player = Player(player_name, room['outside'])
+print('You are entering at your own risk! Type q to quit now!')
+print('Hello', player.name + '!', 'You are in', player.current_room)
+print('Now choose where you would like to explore!')
+directions = {
+    'n': 'North',
+    's': 'South',
+    'e': 'East',
+    'w': 'West',
+    'q': 'Quit'
+}
+
+direction = ''
+
+# Confirms user has not quite and tells the user to choose direction
+# if direction != 'q':
+#     print('Remember you are in the', player.current_room)
+options = ''
+
+for opt_direction in directions:
+    options += f'\n Choose {directions[opt_direction]} enter "{opt_direction}"'
+
+print(options)
+while direction != 'q':
+    direction = input('Which way would you like to go? ')
+    if direction == 'n':
+        print('Lets go North')
+        break
+    elif direction == 's':
+        print('Lets go South')
+        break
+    elif direction == 'e':
+        print('Lets go East')
+        break
+    elif direction == 'w':
+        print('Lets go West')
+        break
+    elif direction == 'q':
+        print('See you later')
+        break
+
```

Comment:
Great work Humberto! The cool thing is , you can add more to the game to make it your very own! :grin: